content/installation/migrating-manager.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bt_wiki
+title: Reinstalling and Migrating Cloudify Manager
+category: Installation
+draft: true
+weight: 400
+
+---
+
+To move your Cloudify Manager to another host, you must use same process as a reinstall. One reason to migrate is to move your Cloudify Manager to a host with more resources. You can do this in one of these ways:
+
+* **In-place reinstall** - Replace Cloudify Manager with the same version on the same host
+* **Migration reinstall** - Replace Cloudify Manager with the same version on another host
+
+The reinstall process includes:
+
+* In-place reinstall -
+
+  1. Backup a snapshot of the Cloudify Manager.
+  1. Backup the agents certificates from the Cloudify Manager.
```

Comment:
I don't believe this step is required- where does it come from?

Unless you mean copying all of the certificates, which is only applicable for a Cloudify 4.0.0 manager on an in-place upgrade.

Reply:
You also missed copying the rest-security.conf from managers prior to 4.2.0.

Reply:
So I should change step 2 from:
Incorrect: Backup the agents certificates from the Cloudify Manager.
Correct: (For upgrade from 4.1.x and below) Backup the rest-security.conf file to a separate location.
Also, what's the path to rest-security.conf?

Reply:
If this is purely for a 4.3 reinstall then neither of those steps are needed.

Doesn't this mean that we're duplicating a bunch of steps though, by having a page specifically for 4.3 and others for others?

Reply:
Also, if this is only for 4.3 to 4.3 reinstall I think a caution note is warranted pointing someone at the migration page because many of the steps are similar so it could lead to harmful confusion.

Reply:
Thanks. Keep in mind that once we have all of the correct information for all 4.3 scenarios, I will reuse sections in the different places so it the same steps in each process are updated together.